python/dask_cudf/dask_cudf/backends.py

```diff
@@ -64,27 +60,34 @@ def _nonempty_series(s, idx=None):
         data = column.build_categorical_column(
             categories=categories, codes=codes, ordered=ordered
         )
-    elif is_string_dtype(dtype):
+    elif is_string_dtype(s.dtype):
         data = ["cat", "dog"]
```

Comment:
Would at the least make this a pyarrow array as it's way more efficient to get it to the GPU, but it would be great if we could generate the data on the GPU instead.